src/modules/Layout/Header/Header.js

```diff
@@ -7,7 +8,12 @@ import './Header.css';
 
 const b = block('j-header');
 
-const Header = ({ logged }) => {
+const propTypes = {
+    logged: PropTypes.bool.isRequired,
```

Comment:
if you already have the user wouldn't that already imply that it is logged in?

Reply:
actually yes, @Vxplus , totally agree!